witness/README.md

```diff
@@ -46,3 +75,8 @@ defined above.  Currently it contains implementations in the following languages
   This can be deployed as a smart contract on the Ethereum blockchain.  There are 
   two main contracts, each of which supports one of the consistency checks 
   described above.
+
+## Feeders
+
+* [Go SumDB](https://github.com/google/trillian-examples/tree/master/sumdbaudit/witness)
```

Comment:
This is potentially confusing, since what's here is a combined feeder and witness.  Maybe just add a little comment pointing that out?